node/src/chain_specs/manta.rs

```diff
@@ -157,32 +167,42 @@ fn manta_dev_genesis(
         sudo: manta_runtime::SudoConfig {
             key: Some(root_key),
         },
+        parachain_staking: ParachainStakingConfig {
+            candidates: invulnerables
+                .iter()
+                .cloned()
+                .map(|(account, _)| {
+                    (
+                        account,
+                        manta_runtime::staking::NORMAL_COLLATOR_MINIMUM_STAKE,
+                    )
+                })
+                .collect(),
+            delegations,
+            inflation_config: manta_runtime::staking::inflation_config::<manta_runtime::Runtime>(),
+        },
         parachain_info: manta_runtime::ParachainInfoConfig {
             parachain_id: MANTA_PARACHAIN_ID.into(),
```

Comment:
The parachain-id needs to change, as this one is already in use.

Reply:
chain spec changes will go into followup PR #926